src/omd.mli

```diff
@@ -55,10 +55,14 @@ type inline =
 type t = inline block list
 (** A markdown document *)
 
+type printer = Html.printer
```

Comment:
Shouldn't we re-export the type here (`type printer = Html.printer = {...}`), since we don't expose the `Html` submodule?